bin/webpack

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
```

Comment:
oh wow you are getting right into it, huh? Awesome!!! 

This may need a little more documentation in the main README since it's some pretty deep config going on in a new workflow. 

Great work!

Reply:
@jywarren Wish I could take credit for being a config ninja, but it's really just the `react-rails` package doing everything behind the scenes.

Take a look at the [docs](https://github.com/reactjs/react-rails#get-started-with-webpacker), most of these config files were generated with just a few `rails generate` type commands.

It's pretty easy to set up in local development, I'm sure it would be easy for you to set this branch up locally on your system. However, it remains to be seen what a production deployment of this would be like.